test/extension/rasterization/Jamfile

```diff
@@ -8,6 +8,10 @@
 #
 import testing ;
 
+compile line.cpp ;
+compile circle.cpp ;
+compile ellipse.cpp ;
+
 run line.cpp ;
 run circle.cpp ;
 run ellipse.cpp ;
```

Comment:
Repeating `compile` and `run` for the same tests is not necessary.
- `compile` is for compile-only tests e.g. static asserts, nothing executed
- `run` is for tests you aim to build and execute